dist/stripe/stripe.js

```diff
@@ -0,0 +1,91 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra-web-app.stripe',
+        [
+            'ngAnimate',
+            'ui.router',
+            'ngMaterial',
+            'pascalprecht.translate',
+            'znk.infra.svgIcon',
+            'znk.infra.popUp',
+            'znk.infra.enum',
+            'znk.infra.config',
+            'znk.infra.storage',
+            'znk.infra.auth',
+            'znk.infra.analytics'
+        ]);
+})(angular);
+
+
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra-web-app.stripe')
+        .service('StripeService',
+            ["$log", "$q", "$window", "$filter", "$http", "AuthService", "ENV", function ($log, $q, $window, $filter, $http, AuthService, ENV) {
+                'ngInject';
+
+                const stripeToken = ENV.stripeToken;
+                const translate = $filter('translate');
+                const paymentApi = `${ENV.znkBackendBaseUrl}/payment`;
+
+                /**
+                 * Open stripe payment modal foe instant payment.
+                 * @returns StripeResult object{Subject<boolean>}
+                 *
+                 * stripe test card 4242 4242 4242 4242
+                 */
+                this.openStripeModal = (amount, name, description, image) => {
+                    let tokenId = null;
+                    const defer = $q.defer();
+                    const handler = ($window).StripeCheckout.configure({
+                        key: stripeToken,
+                        amount: amount ? amount * 100 : 0, // amount to display: ex: $20 * 100 === $20.00
```

Comment:
I think you should have another var amountIn cents....

Reply:
fixed